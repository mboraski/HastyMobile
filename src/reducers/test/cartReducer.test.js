// import { addToCart, removeFromCart, setCurrentLocation } from '../../actions/cartActions';
// import reducer, { initialState } from '../cartReducer';
//
// describe('cartReducer', () => {
//     it('handles addToCart action', () => {
//         const product = {
//             quantity: 1,
//             title: 'Type 1 code 1',
//             thumbnail_image: 'image',
//             price: '$3.49',
//             productCode: '1',
//             deliveryType: '1'
//         };
//         expect(reducer(initialState, addToCart(product))).toMatchSnapshot();
//     });
//     it('handles removeFromCart action', () => {
//         const product = {
//             quantity: 1,
//             title: 'Type 1 code 1',
//             thumbnail_image: 'image',
//             price: '$3.49',
//             productCode: '1',
//             deliveryType: '1'
//         };
//         expect(reducer(initialState, addToCart(product))).toMatchSnapshot();
//     });
//     it('handles addToCart then removeFromCart', () => {
//         const product = {
//             quantity: 1,
//             title: 'Type 1 code 1',
//             thumbnail_image: 'image',
//             price: '$3.49',
//             productCode: '1',
//             deliveryType: '1'
//         };
//         const newState = reducer(initialState, addToCart(product));
//         expect(reducer(newState, removeFromCart(product))).toMatchSnapshot();
//     });
//     it('handles setCurrentLocation', () => {
//         expect(
//             reducer(initialState, setCurrentLocation('address', { latitude: 0, longitude: 1 }))
//         ).toMatchSnapshot();
//     });
// });
