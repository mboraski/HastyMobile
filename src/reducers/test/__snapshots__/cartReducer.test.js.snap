// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cartReducer handles addToCart action 1`] = `
Object {
  "currentSetAddress": "",
  "currentSetLatLon": Object {
    "lat": null,
    "lon": null,
  },
  "orderId": "",
  "products": Object {
    "1": Object {
      "1": Object {
        "deliveryType": "1",
        "price": 3.49,
        "productCode": "1",
        "quantity": 1,
        "thumbnail_image": "image",
        "title": "Type 1 code 1",
      },
    },
  },
  "totalCost": 3.49,
  "totalQuantity": 1,
}
`;

exports[`cartReducer handles addToCart then removeFromCart 1`] = `
Object {
  "currentSetAddress": "",
  "currentSetLatLon": Object {
    "lat": null,
    "lon": null,
  },
  "orderId": "",
  "products": Object {
    "1": Object {
      "1": Object {
        "deliveryType": "1",
        "price": 3.49,
        "productCode": "1",
        "quantity": 0,
        "thumbnail_image": "image",
        "title": "Type 1 code 1",
      },
    },
  },
  "totalCost": 0,
  "totalQuantity": 0,
}
`;

exports[`cartReducer handles removeFromCart action 1`] = `
Object {
  "currentSetAddress": "",
  "currentSetLatLon": Object {
    "lat": null,
    "lon": null,
  },
  "orderId": "",
  "products": Object {
    "1": Object {
      "1": Object {
        "deliveryType": "1",
        "price": 3.49,
        "productCode": "1",
        "quantity": 1,
        "thumbnail_image": "image",
        "title": "Type 1 code 1",
      },
    },
  },
  "totalCost": 3.49,
  "totalQuantity": 1,
}
`;

exports[`cartReducer handles setCurrentLocation 1`] = `
Object {
  "currentSetAddress": "address",
  "currentSetLatLon": Object {
    "lat": 0,
    "lon": 1,
  },
  "orderId": "",
  "products": Object {},
  "totalCost": 0,
  "totalQuantity": 0,
}
`;
